staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -236,8 +236,9 @@ const (
 // executing at each point where there is a change in that quantity,
 // because the metrics --- and only the metrics --- track that
 // quantity per FlowSchema.
-func (qs *queueSet) StartRequest(ctx context.Context, hashValue uint64, flowDistinguisher, fsName string, descr1, descr2 interface{}, queueNoteFn fq.QueueNoteFn) (fq.Request, bool) {
-	// all request(s) have a width of 1, in keeping with the current behavior
+func (qs *queueSet) StartRequest(ctx context.Context, _ float64, hashValue uint64, flowDistinguisher, fsName string, descr1, descr2 interface{}, queueNoteFn fq.QueueNoteFn) (fq.Request, bool) {
```

Comment:
BTW - can you also address this issue as part of this PR:
https://github.com/kubernetes/kubernetes/issues/102299

?

Reply:
@wojtek-t  actually I am going through https://github.com/kubernetes/kubernetes/issues/102299 now and have a few questions, is it okay if I do it in a follow up PR?

Reply:
OK - sure